deps.edn

```diff
@@ -1,4 +1,4 @@
-{:paths ["src"]
+{:paths ["src/cljs"]
```

Comment:
might be good to also update the test `extra-path` as those are also nested under a `cljs` directory.